unit_tests/FileTests.d

```diff
@@ -149,3 +149,14 @@ unittest
         "test_files/variables.d",
         "test_files/variables.h");
 }
+
+unittest
+{
+    assertRunsDStepCFile(
+        "test_files/aggregate.d",
+        "test_files/aggregate.h");
+
+    assertRunsDStepObjCFile(
+        "test_files/objc/methods.d",
+        "test_files/objc/methods.h");
```

Comment:
These tests are more like integration tests. Perhaps they should have a separate directory?